input/projects/data/LTAF.md

```diff
@@ -0,0 +1,20 @@
+---
+Title: Lightweight Test Automation Framework [Archived]
+Contributor: Outercurve
+Logo: outercurve.png
+Web: https://archive.codeplex.com/?p=ltaf
+---
```

Comment:
Let's add `IsActive: false` to this top section here and to other projects that are no longer maintained (Legacy SourceLink, Nancy, etc). That way we can split them out on the project list.

Reply:
This change has been made - should be ready to go